src/analytics.ts

```diff
@@ -57,6 +57,7 @@ export function createAnalyticsInstance(options?: Options) {
                         device_type: growthbookOptions?.attributes.device_type,
                     }),
                     ...(growthbookOptions?.attributes?.url && { url: growthbookOptions?.attributes.url }),
+                    ...(growthbookOptions?.attributes?.user_ip && { url: growthbookOptions?.attributes.user_ip }),
```

Comment:
This can be fall under privacy policy
```
As such, organizations processing IP addresses must adhere to the strict regulations on data protection and privacy. Under the GDPR, when an IP address is collected and processed, it is treated with the same level of protection as any other personal data, such as names, addresses, or social security numbers.
```
Need to confirm with the compliance before we proceed with this.

Reply:
cloudflare by default also collect user ip address